pkg/kubectl/cmd/util/factory.go

```diff
@@ -63,6 +63,10 @@ const (
 	FlagMatchBinaryVersion = "match-server-version"
 )
 
+var (
+	FlagHTTPCacheDir = "cachedir"
```

Comment:
cc @kubernetes/sig-cli-api-reviews for global flag addition

Reply:
Thanks.  ACK.